packages/packagers/js/src/DevPackager.js

```diff
@@ -116,7 +116,7 @@ export class DevPackager {
         let output = code || '';
         wrapped +=
           JSON.stringify(this.bundleGraph.getAssetPublicId(asset)) +
-          ':[function(require,module,exports) {\n' +
+          ':[function(require,module,exports,__globalThis) {\n' +
```

Comment:
Where will that `__globalThis` be referenced?

Reply:
It is referenced [here](https://github.com/parcel-bundler/parcel/blob/v2/packages/transformers/js/core/src/global_replacer.rs#L139-L154)

I should have added it to the description, my bad :)

Reply:
that's referenced via `arguments[3]` though. Wouldn't this mean that if someone declared `let __globalThis` in their code they'd get an error stating that it is already defined? Doesn't look like this variable is used directly.

Reply:
Yes, that's correct. I opted for `__globalThis` since `__global` is more likely to be defined. I'd prefer to use `globalThis` directly instead of specifying an argument since it's well supported, but it would be inconsistent with the parcel runtime.